test/e2e/suite/carotation/carotation.go

```diff
@@ -37,8 +37,11 @@ const (
 	mountPath     = "/var/run/secrets/my-pod"
 	containerName = "my-container"
 
-	pollInterval = 1 * time.Second
-	pollTimeout  = 60 * time.Second
+	// pollInterval is set to 5s here because the updateRetryPeriod set when the driver
+	// starts the camanager is 5s. Polling every second is wasteful because updates will
+	// only be made every 5 seconds.
+	pollInterval = 5 * time.Second
+	pollTimeout  = 300 * time.Second
```

Comment:
note: I increased the polling timeout because updates can only happen every 5 seconds due to the update period given by the driver when the camanager is started. That means there are only a maximum of 12 possible polls which could see a difference, and local testing indicated that that was too restrictive and the cause of many flakes.

I've seen far fewer flakes with this setup. Maybe a future PR should reduce the update period and the poll interval here.